distribution/ddf-common/src/main/resources/etc/org.apache.karaf.command.acl.ssh.cfg

```diff
@@ -0,0 +1,5 @@
+#
+# This configuration file defines the ACLs for commands in the ssh subshell
+#
+
+* = NO_ACCESS
```

Comment:
Is this what restricts ssh access to only the localhost?

Reply:
No. This is what prevents any outbound ssh calls from the karaf shell.

Reply:
To clarify what it does, this sets access to all the ssh subshell commands to users with the ACL `NO_ACCESS`.